docs/tracks.md

```diff
@@ -0,0 +1,174 @@
+# Video, Audio and Text Tracks
+
+The Kaltura Player uses a comprehensive API to handle all kind of tracks, including video (bitrate), audio, and text (language) tracks.
+<br>This document shows how to use the API to implement and manage player tracks. 
```

Comment:
Is this ok in terms of my change?

Reply:
I'm not sure about 'implement'. the API is for managing tracks not for implementing.